docs/installing-northstar/basic-setup.md

```diff
@@ -4,22 +4,25 @@
 
 {% embed url="https://www.youtube.com/watch?v=bK4pV-AHOHM" %}
 
-{% hint style="info" %}
-TODO: Method to get to Origin's local files described below is incorrect/doesn't exist
-{% endhint %}
-
-Firstly note that Northstar client is only available on PC and requires you to **both own the game and have it installed**.
+Firstly, note that the Northstar client is only available on PC and requires you to **both own the game and have it installed**.
 
 1. Download the latest version of Northstar from the [releases](https://github.com/R2Northstar/Northstar/releases) page
-2. Copy all the files in the newly downloaded zip folder to your Titanfall folder
-   * **For Steam** - Right click _Titanfall 2_ > Open _Properties_ > Click _Local Files_ > Click _Browse_
-   * **For Origin** - Click Titanfall 2 in the Library > Click the gear icon > Click _Game Properties_
+2. Copy all the files in the newly downloaded zip folder to your Titanfall2 folder. This can vary depending on whether you purchased the game off Steam or Origin, and if you set a custom folder for game installs.
+   ```
+   examples:
+      C:\Program Files (x86)\Origin Games\Titanfall2
+      C:\Program Files (x86)\Steam\steamapps\common\Titanfall2
+      D:\MyCustomGameFolder\Titanfall2
+   ```
+   {% hint style="info" %}
+   For those on Steam, you don't have to go folder hunting! In your Steam library, right click _Titanfall 2_ > Open _Properties_ > Click _Local Files_ > Click _Browse_
+   {% endhint %}
```

Comment:
I'd argue going via game store UI to find the folder should be the preferred method to prevent situations like "I can't find my install location!!1!!11!".
It's annoying that Origin doesn't have the same feature like Steam where you can just open the folder a game is in by selecting it in the game library.